inc/functions/carrier.php

```diff
@@ -6,41 +6,39 @@
  */
 
 use BracketSpace\Notification\Interfaces;
-
+use BracketSpace\Notification\Defaults\Store\Carrier as CarrierStore;
 /**
  * Registers Carrier
  *
- * @uses notification/carriers filter
- * @since  6.0.0
+ * @since  [Next] Use Carrier Store.
```

Comment:
The comment is ok here, but you shouldn't remove the previous one. When you are changing something to a function or method you are *adding* the `@since` tag, not replacing it